docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,206 @@
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 8080
+  selector:
+    app: screwdriver
+    tier: api
+```
+
+#### Get your Kubernetes token name
+Kubernetes actually sets up your Kubernetes token by default. You will need this for your `deployment.yaml`.
+Kubectl can be used to see your [Kubernetes secrets](http://kubernetes.io/docs/user-guide/secrets/walkthrough/).
+
+- Get the `<DEFAULT_TOKEN_NAME>`, by running:
+```bash
+$ kubectl get secrets
+
+NAME                      TYPE                                  DATA      AGE
+default-token-abc55       kubernetes.io/service-account-token   3         50d
+```
+
+The `<DEFAULT_TOKEN_NAME>` will be listed under `Name` when the `Type` is `kubernetes.io/service-account-token`.
+
+
+#### Create a Deployment
+A Deployment makes sure a specified number of pod “replicas” are running at any one time. If there are too many, it will kill some; if there are too few, it will start more. Follow instructions on the [Deploying Applications](http://kubernetes.io/docs/user-guide/deploying-applications/) page to create your `deployment.yaml`.
+
+It should look similar to the following:
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sdapi
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: screwdriver
+        tier: api
+    spec:
+      containers:
+      - name: screwdriver-api
+        image: screwdrivercd/screwdriver
+        ports:
+        - containerPort: 8080
+        # Environment variables that point to your secrets in your secrets.yaml
+        env:
+          - name: DATASTORE_PLUGIN
+            value: dynamodb
+          - name: DATA_DIR
+            value: /opt/screwdriver/data
+          - name: SECRET_JWT_PRIVATE_KEY
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: jwtprivatekey
+          - name: SECRET_OAUTH_CLIENT_ID
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: oauthclientid
+          - name: SECRET_OAUTH_CLIENT_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: oauthclientsecret
+          - name: SECRET_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: password
+          - name: K8S_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: <DEFAULT_TOKEN_NAME>     # insert your K8s token name here
+                key: token
+          - name: DATASTORE_DYNAMODB_ID
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: dynamodbid
+          - name: DATASTORE_DYNAMODB_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: dynamodbsecret
+```
+
+#### Deploy
+For a fresh deployment, run the `kubectl create` command on your `yaml` files:
```

Comment:
It may be worth mentioning something like:

First create the service, then create the deployment.